src/controller.financial.js

```diff
@@ -33,8 +31,17 @@ module.exports = function(Chart) {
 					var h = data.datasets[tooltipItem.datasetIndex].data[tooltipItem.index].h;
 					var l = data.datasets[tooltipItem.datasetIndex].data[tooltipItem.index].l;
 					var c = data.datasets[tooltipItem.datasetIndex].data[tooltipItem.index].c;
-
-					return ' O ' + o + ' H ' + h + ' L ' + l + ' C ' + c;
+					
+					var f = data.datasets[tooltipItem.datasetIndex].tooltipLength;
+					if(f !== undefined) {
+						o = o.toFixed(f);
+						h = h.toFixed(f);
+						l = l.toFixed(f);
+						c = c.toFixed(f);
+					}
+
+					//return ` O ${o} H ${h} L ${l} C ${c}`;
+					return ' O: ' + o + '    H: ' + h + '    L: ' + l + '    C: ' + c;
```

Comment:
@benmccann Are template literals and other ES6 features allowed ?
If yes, we'd need to replace gulp-uglify with gulp-uglify-es ?

Reply:
Chart.js supports IE11, so it'd make sense for us to support it as well, which means we'd have to transpile to es5. I'm not sure it's worth the complexity of the extra build step. I'd probably just suggest removing the commented out line above

Reply:
Removed